env.md

```diff
@@ -0,0 +1,118 @@
+# Inventory of Agoric SDK Environment Variables
+
+```js
+process.env.DOCUMENTATION // for power users
+```
+
+TODO: Fill out more details!
+
+This file describes environment variables that influence the execution of the
+Agoric SDK.  They are documented here as an advanced feature to be kept
+up-to-date with the source code.
+
+## Template
+
+**Add to a new section in alphabetical order.**
+
+Affects:
+
+Purpose:
+
+Description:
+
+Lifetime:
+
+## ALLOW_IMPLICIT_REMOTABLES
+
+Affects: marshal
```

Comment:
Here's hoping (in a later PR) to refine these "Affects" targets to be in our reader's vocabulary. Putting "marshal" in our docs search box turns up no hits. The closest I see is [Far\(\) and Remotable Objects \| Beta](https://agoric.com/documentation/guides/js-programming/far.html#far-and-remotable-objects).